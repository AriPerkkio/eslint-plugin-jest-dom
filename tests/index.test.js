const { rules } = require('../lib/index');

it('should have all the rules', () => {
  expect(rules).toMatchSnapshot();
});
