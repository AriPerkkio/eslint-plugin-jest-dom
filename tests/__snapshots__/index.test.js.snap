// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have all the rules 1`] = `Object {}`;
